src/hashcat.c

```diff
@@ -2895,7 +2896,7 @@ static int run_kernel (const uint kern_run, hc_device_param_t *device_param, con
     {
       if (data.opti_type & OPTI_TYPE_SLOW_HASH_SIMD)
       {
-        num_elements = CEIL ((float) num_elements / device_param->vector_width);
+        num_elements = CEIL (num_elements / device_param->vector_width);
```

Comment:
Not really. I added a printf to see if anything changed after I removed the cast. Numbers were still the same.

The only issue I can potentially see is that dividing could cause a remainder to get thrown away. num_elements seems to end up being a multiple of 8 though.